src/main/java/frc/trigon/robot/subsystems/ledstrip/LEDStrip.java

```diff
@@ -32,158 +27,76 @@ public class LEDStrip extends SubsystemBase {
-                        inverted,
-                        this.offset
-                ),
-                animationSlot
-        );
-    }
-
-    void animateRainbow(double brightness, double speed) {
-        CANDLE.animate(
-                new RainbowAnimation(
-                        brightness,
-                        speed,
-                        this.numberOfLEDs,
-                        inverted,
-                        this.offset
-                ),
-                animationSlot
-        );
+    private void setLedColors(edu.wpi.first.wpilibj.util.Color color, int index) {
+        LEDStripConstants.LED_BUFFER.setLED(index, color);
+        LED.setData(LEDStripConstants.LED_BUFFER);
     }
 
-    void animateColorFlow(Color color, double speed) {
-        CANDLE.animate(new ColorFlowAnimation(
-                        color.getRed(),
-                        color.getGreen(),
-                        color.getBlue(),
-                        0,
-                        speed,
-                        this.numberOfLEDs,
-                        inverted ? ColorFlowAnimation.Direction.Backward : ColorFlowAnimation.Direction.Forward,
-                        this.offset
-                ),
-                animationSlot
-        );
+    private void setAllLedColors(Color color) {
+        for (int index = 0; index < numberOfLEDs; index++) {
+            setLedColors(color, index);
+        }
     }
 
-    void animateLarson(Color color, double speed, LarsonAnimation.BounceMode mode, int size) {
-        CANDLE.animate(
-                new LarsonAnimation(
-                        color.getRed(),
-                        color.getGreen(),
-                        color.getBlue(),
-                        0,
-                        speed,
-                        this.numberOfLEDs,
-                        mode,
-                        size,
-                        this.offset
-                ),
-                animationSlot
-        );
+    void clearLedColors() {
+        setAllLedColors(Color.kBlack);
     }
 
-    void animateRGBFade(double brightness, double speed) {
-        CANDLE.animate(
-                new RgbFadeAnimation(
-                        brightness,
-                        speed,
-                        this.numberOfLEDs,
-                        this.offset
-                ),
-                animationSlot
-        );
+    void staticColor(Color color) {
+        setAllLedColors(color);
     }
 
-    void animateSingleFade(Color color, double speed) {
-        CANDLE.animate(
-                new SingleFadeAnimation(
-                        color.getRed(),
-                        color.getGreen(),
-                        color.getBlue(),
-                        0,
-                        speed,
-                        this.numberOfLEDs,
-                        this.offset
-                ),
-                animationSlot
-        );
+    void blink(Color color) {
```

Comment:
`toggleBlink`. Also why is this toggle?

Reply:
I call it periodically in the commands. Why shouldn't it be toggle?

Reply:
Oh, I see now. I thought it was toggling between blinking and being off. Maybe change the variable name to `areLEDsOn` or something like that.